bumble/gatt_server.py

```diff
@@ -79,6 +80,45 @@ def get_attribute(self, handle):
                 return attribute
         return None
 
+    def get_service_attribute(self, service_uuid: UUID) -> Service:
+        for attribute in self.attributes:
+            if attribute.type == GATT_PRIMARY_SERVICE_ATTRIBUTE_TYPE:
+                if attribute.uuid == service_uuid:
+                    return attribute
+        raise NotFoundError()
```

Comment:
would be better to return `None` when not found, to avoid forcing the caller to use a `try`.